phpunit.xml

```diff
@@ -0,0 +1,29 @@
+<phpunit
+        backupGlobals="false"
+        backupStaticAttributes="false"
+        bootstrap="bootstrap.php"
+        convertErrorsToExceptions="true"
+        convertNoticesToExceptions="true"
+        convertWarningsToExceptions="true"
+        colors="true">
+    <php>
+        <ini name="error_reporting" value="-1"/>
```

Comment:
I'm not sure about this one. I'd not suppress errors completely as this is also a good indicator that some parts of the code may be deprecated or invalid.

Reply:
Of course I can change it, but in ez packages and symfony, in phpunit.xml files is the same config

Reply:
I think we should remove this line.

Reply:
OK, no problem :)